Formula/unittest-cpp.rb

```diff
@@ -12,13 +12,12 @@ class UnittestCpp < Formula
     sha256 "67f3d01e1d608e9f27d230e8e2bb35851248a69d526f120e78b13d9fdc2874b4" => :mavericks
   end
 
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
-  depends_on "libtool" => :build
-
   def install
-    system "autoreconf", "-fvi"
     system "./configure", "--prefix=#{prefix}", "--disable-silent-rules"
     system "make", "install"
   end
+
+  test do
+    assert_match version.to_s, shell_output("cat /usr/local/lib/pkgconfig/UnitTest++.pc | grep Version: | cut -d' ' -f2")
```

Comment:
You don't need to use shell utilities directly here, Ruby can manage this itself:

```ruby
assert_match version.to_s, File.read(lib/"pkgconfig/UnitTest++.pc")
```

It'd be nice to write a more comprehensive test though, if you can think of one.